source/includes/v1/query_language_and_geo_filtering.md

```diff
@@ -139,6 +145,7 @@ Function name | Description
 `now`         | Return the current date. This function should be called as a query value for a field
 `null`        | Search for records where no value is defined for the given field
 `exact`       | Search for records with a field exactly matching a given value
+`search`      | Search for records with one or more fields containing a given value
```

Comment:
I think that using "containing" is misleading. 

For example, https://documentation-resources.opendatasoft.com/explore/dataset/les-arbres-remarquables-de-paris/table/?q=%23search(adresse,+%22yramide%22) doesn't return any value.

It could be something like "search for records with one or more fields with a word starting with a given value"?

Reply:
We could also talk about the lack of case sensitivity.